apps/postgres_operator.py

```diff
@@ -28,5 +38,33 @@ def objects():
             },
             "numberOfInstances": 1,
             "postgresql": {"version": "14"},
+            # https://thedatabaseme.de/2022/03/26/backup-to-s3-configure-zalando-postgres-operator-backup-with-wal-g/
+            "env": list(map(lambda pair: {"name": pair[0], "value": pair[1]}, {
+                "WAL_S3_BUCKET": "ocf-postgres-backup",
+                "WAL_BUCKET_SCOPE_PREFIX": "",
+                "WAL_BUCKET_SCOPE_SUFFIX": "",
+                "USE_WALG_BACKUP": "true",
+                "USE_WALG_RESTORE": "true",
+                'BACKUP_SCHEDULE': '00 10 * * *',
+                "AWS_S3_FORCE_PATH_STYLE": "true",
+                "AWS_ENDPOINT": "https://o3.ocf.berkeley.edu",
+                "AWS_REGION": "us-east-1",
```

Comment:
See https://github.com/ocf/outline/blob/e915b129acdc94523d4f65c3bb0cc65ba9d5a10b/.transpire.py#L62-L67 for how to set this up!